lib/file_store/s3_store.rb

```diff
@@ -111,6 +112,12 @@ def store_file(file, path, opts = {})
       [File.join(absolute_base_url, path), etag]
     end
 
+    def delete_file(path)
```

Comment:
```suggestion
    def delete_object(path)
```

Personally I prefer that the method name is similar to the API name